### ISSUE TITLE
Fail Clone Github Repo

### ISSUE THREAD

**Issue by FireBladeNooT** [BenjV/SYNO-packages#38]
Hi,
I've got an error when I manualy install **"Medusa DSM 7 V2.1.spk"** (the good one this time) on DSM 7 (ds414j):
_Failed to clone the github repositary: https://github.com/pymedusa/Medusa.git_

Git package is installed and run properly (2.33.0-19).
(repositary is not a mistake, it was written like that in message).

NB: It say version 2.0 installed even with 2.1 psk file

**Comment by BenjV** [comment]
I forgot to update the version number within the package, I have  correct so if you wish you could download a fresh copy.
But that has no influence on the working of the package.
That src folder is created by the git clone command, so if that failed the folder is not created.

I can only think of two reasons.
1. Git is incorrect working/installed.
2. A network problem so the medusa github could not be reached. 

I tested the package and lots of other people installed it with no problem.

**Comment by FireBladeNooT** [comment]
OK, I'll keep investigate on this way

**Comment by GeertLitjens** [comment]
Not sure if this is the problem, but when I open the SPK file, the git command has --quit whereas it should be --quiet. When copying and executing the command git fails because --quit is unsupported.

**Comment by BenjV** [comment]
Oeps typo in the latest update.
I corrected it and if you download the latest package it should install fine.

**Comment by FireBladeNooT** [comment]
With trying manual git clone it says version `GLIBC_2.25' not found and I found my version is 2.20
I'll try to update it

**Comment by FireBladeNooT** [comment]
It seems very complicate to update to GLIBC_2.25 with DS214j as it is not provided by Synology.
So I look avoid using of GIT and I've succed to install medusa, but i've got to modify your package to not git clone but copy a local repo of pymedus/medusa on my NAS.

Maybe it could be ok to made an spk which ask a local repo ?
Here the postinst I've made to install medusa, it is not perfect but the idea is here.
A good version of spk will remove git dependancy, and also not use Python 3.8 of community as it required GLIBC_2.25.

[postinst_Custom.txt](https://github.com/BenjV/SYNO-packages/files/7231699/postinst_Custom.txt)
